app/templates/package.json

```diff
@@ -23,27 +23,29 @@
   },
   "homepage": "https://github.com/<%= user %>/<%= repo %>",
   "devDependencies": {
-    "babel-core": "^5.2.17",
-    "babel-eslint": "^4.0.5",
-    "babel-loader": "^5.3.2",
-    "chai": "^3.2.0",
+    "babel-core": "^6.3.26",
```

Comment:
Thanks @jmm didn't knkw it was deprecated
On Dec 31, 2015 11:17 AM, "James, please" <notifications@github.com> wrote:

> In app/templates/package.json
> <https://github.com/babel/generator-babel-boilerplate/pull/299#discussion_r48661696>
> :
>
> > @@ -23,27 +23,29 @@
> >    },
> >    "homepage": "https://github.com/<%= user %>/<%= repo %>",
> >    "devDependencies": {
> > -    "babel-core": "^5.2.17",
> > -    "babel-eslint": "^4.0.5",
> > -    "babel-loader": "^5.3.2",
> > -    "chai": "^3.2.0",
> > +    "babel-core": "^6.3.26",
>
> Makes sense. I'll change this back, then.
>
> Thanks @jmm <https://github.com/jmm>!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/babel/generator-babel-boilerplate/pull/299/files#r48661696>
> .
>